Fix block explorer guides (#386)
matter-labs/zksync-web-era-docs: docs/api/tools/block-explorer/batches-view.md

# Browse batches

> Batches are like groups of transactions that are processed together in one [block](../../../dev/developer-guides/transactions/blocks.md#blocks) before they're added to the main blockchain. It increases the transaction throughput of zkSync Era and reduces its transaction fees.

Reviewer: ```suggestion
Batches are groups of transactions that are processed together in one [block](../../../dev/developer-guides/transactions/blocks.md#blocks) before they're added to the main blockchain. Batches increase the transaction throughput of zkSync Era and reduce transaction fees.
```

Reviewer: I removed the final sentence because it is repeated in the next para.

> By putting several transactions into a single batch, we reduce the number of transactions that need to be processed and validated, this results in faster transaction processing times and lower fees for users interacting with zkSync Era.

Reviewer: ```suggestion
By putting multiple transactions in a single batch, we reduce the number of transactions that need to be processed and validated. This results in faster transaction processing times and lower fees for users interacting with zkSync Era.
```

> ![Browse batches!](../../../assets/images/batches.png "View batches on zkSync")

Reviewer: This should be of the Block Explorer home screen, or dashboard. This picture here is for the next section.

> On the block explorer home screen, you'll see the 10 most recent batches by default. If you want to see all the recent batches or get details on a specific batch, you can go to the batches page, which is also accessible from the top menu.

Reviewer: ```suggestion
On the block explorer home screen, you see the 10 most recent batches by default. If you want to see all recent batches, or details on a specific batch, go to the batches page which is accessible from the top menu.
```

Reviewer: Latest batches is clear in the dashboard. Use a full dashboard pic here with arrow.

## See all batches

> You can view the details of a specific batch by clicking on its batch number. To explore more batches, go to the "[Batches](https://explorer.zksync.io/batches/)" section in the navigation menu and use the pagination to navigate to the next page. 

Reviewer: ```suggestion
View the details of a specific batch by clicking on its batch number on the [Batches](https://explorer.zksync.io/batches/) page.

To see more batches, use the pagination at the bottom to navigate through the pages. 
```

> If you've left the homepage, you can still search for a batch by typing in its batch number. 

Reviewer: ```suggestion
Use the search function on the explorer dashboard to search for a batch by its batch number. 
```

Reviewer: Add a pic of the search bar here.

> Batches are similar to blocks and have properties like block headers and transactions.

Reviewer: I don't see **block header** anywhere in batches?

1. The block header

> - This shows brief information about this specific batch, as well as a link to the previous batch in the chain.

Reviewer: I don't know what you are referring to. Can you add a pic if it exists?

2. Transactions

> - Lists of all the transactions that were included in this batch.

Reviewer: ```suggestion
- A list of all the transactions included in the batch.
```

![Batches transactions!](../../../assets/images/batches-trx.png "View batches transactions")

> Check for the [block header](./block-view.md#block-headers) and [transaction](./block-view.md#transactions) batch properties.

Reviewer: You link to the Block details but this is different to Batches. Can you clarify what you mean please?
